mappings/net/minecraft/achievement/Achievement.mapping

```diff
@@ -1 +1,27 @@
 CLASS net/minecraft/class_24 net/minecraft/achievement/Achievement
+	FIELD field_1202 xLocation I
+	FIELD field_1203 yLocation I
+	FIELD field_1204 parent Lnet/minecraft/class_24;
+	FIELD field_1205 achievementIcon Lnet/minecraft/class_31;
+	FIELD field_1206 description Ljava/lang/String;
+	FIELD field_1207 challenge Z
+	FIELD field_1208 formattedString Lnet/minecraft/class_241;
+	METHOD <init> (ILjava/lang/String;IILnet/minecraft/class_124;Lnet/minecraft/class_24;)V
+		ARG 1 id
+		ARG 2 descriptionText
+		ARG 3 xLocation
+		ARG 4 yLocation
+	METHOD <init> (ILjava/lang/String;IILnet/minecraft/class_17;Lnet/minecraft/class_24;)V
+		ARG 1 id
+		ARG 2 descriptionText
+		ARG 3 xLocation
+		ARG 4 yLocation
+	METHOD <init> (ILjava/lang/String;IILnet/minecraft/class_31;Lnet/minecraft/class_24;)V
+		ARG 1 id
+		ARG 2 descriptionText
+		ARG 3 xLocation
+		ARG 4 yLocation
+	METHOD method_1040 challenge ()Lnet/minecraft/class_24;
+	METHOD method_1042 formatter (Lnet/minecraft/class_241;)Lnet/minecraft/class_24;
```

Comment:
Are these getters? Getters have a get prefix

Reply:
They are builders.

Reply:
Ah, I didn’t look close enough. That’s right then